src/main/java/bio/terra/cli/app/utils/AppDefaultCredentialUtils.java

```diff
@@ -19,19 +19,7 @@
 
   /** Return the absolute path to file backing the current application default credentials. */
   public static Optional<Path> getADCBackingFile() {
```

Comment:
nit: probably rename this `getAdcBackingFile()` to match convention.

Reply:
Hmm, this class has 3 `ADC`, 1 `Adc`, and 1 `ApplicationDefaultCredentials`. I think `ADC` is ok.